### ISSUE TITLE
test set not in zurich-raw-to-rgb.zip

### ISSUE THREAD

**Issue by aRrtTist** [goutamgmb/deep-burst-sr#11]
Hi,
    I can't find the test dataset in the zip from https://data.vision.ee.ethz.ch/bhatg/zurich-raw-to-rgb.zip, there is train dataset only, so how can I get the test dataset?
![image](https://user-images.githubusercontent.com/43052152/142171253-2ee08c0e-c6f0-4b84-8549-9eceeac460b6.png)

**Comment by goutamgmb** [comment]
Hi,

Which test set do you refer to? Note that for experiments in our paper, we used the "SyntheticBurst validation set" listed [here](https://github.com/goutamgmb/deep-burst-sr#datasets).
